### PR TITLE
Update SellJunk.lua to use C_Container where needed

### PR DESCRIPTION
**Pull request by redpancho**
This updates SellJunk.lua to work on Beta. Converted some magic numbers to constant and using the namedspaced C_Container functions where necessary

### CODE REVIEW

**Comment by lure** [comment]
Are you sure it works on current pre-patch?

**Comment by redpancho** [comment]
> Are you sure it works on current pre-patch? On current release
> 
> ```
> Message: [string "return C_Container.GetContainerNumSlots(1)"]:1: attempt to call field 'GetContainerNumSlots' (a nil value)
> Time: Wed Nov  9 20:56:23 2022
> ```

Updated to work in both versions now

File: SellJunk.lua [hunk 1]
```diff
@@ -31,12 +36,19 @@ SellButton:SetScript("OnClick", function()
...
+
+			if (link) then
 				_, _, quality, _, _, _, _, _, _, _, vendorPrice = GetItemInfo(link)
 			end
 
-			if (link and quality == 0) then
+			if (link and quality == Enum.ItemQuality.Poor) then
+				local itemInfo, count = GetContainerItemInfo(container, slot)
+
+				if (type(itemInfo) == 'table') then
```
[the remark this answers is not in the document]
Comment (redpancho) [reply SellJunk.lua:48]: What would you like to see to have this pull request accepted or closed?

I'm not sure I understood your message unfortunately. I tested this code on both Retail and Beta DF and it worked on both.

**Comment by lure** [comment]
Thank you for your work!

**Pull request merged**